Makefile

```diff
@@ -28,6 +28,10 @@ else    # $(MSBUILD) != 1
 _SLN_BUILD  = MSBUILD="$(MSBUILD)" tools/scripts/xabuild
 endif   # $(USE_MSBUILD) == 1
 
+ifneq ($(IGNORE_MAX_MONOVERSION),)
```

Comment:
I don't understand the use case for this. Nobody is going to want to remember to run `make IGNORE_MAX_MONOVERSION=1`; it should instead be specified in `Configuration.Override.props`.